WcaOnRails/db/migrate/20170517174106_add_report_nag_sent_at_to_competitions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddReportNagSentAtToCompetitions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :Competitions, :report_nag_sent_at, :datetime
```

Comment:
I don't think join could help, see [this weird thing](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/models/competition.rb#L346-L350).